packages/compass-e2e-tests/tests/collection-rename.test.ts

```diff
@@ -6,16 +6,15 @@ import {
   screenshotIfFailed,
   skipForWeb,
   TEST_COMPASS_WEB,
-  DEFAULT_CONNECTION_NAME,
+  DEFAULT_CONNECTION_NAME_1,
 } from '../helpers/compass';
 import type { CompassBrowser } from '../helpers/compass-browser';
-import { createBlankCollection, dropDatabase } from '../helpers/insert-data';
+import { createDummyCollections } from '../helpers/insert-data';
 import * as Selectors from '../helpers/selectors';
 
-const initialName = 'numbers';
-const newName = 'renamed';
-
-const databaseName = 'rename-collection';
+const databaseName = 'test';
```

Comment:
I changed this file to just use the same test database we use everywhere so it will be cleaned up between tests in the standard way along with everything else. No reason for a new database to test renaming a collection.